tools/fragpipe/macros.xml

```diff
@@ -0,0 +1,1206 @@
+                #set $cxt = $prfx.ionquant.labeling_conditional.lfq
+                #set $wfdict['ionquant.use-labeling'] = 'false'
+                #set $wfdict['ionquant.use-lfq'] = 'true'
+                #if $cxt.maxlfq != 'None'
+                    #set $wfdict['ionquant.maxlfq'] = $cxt.maxlfq
+                #end if
+                #if $cxt.minions != ''
+                    #set $wfdict['ionquant.minions'] = $cxt.minions
+                #end if
+            #end if
+            #set $cxt = $prfx.ionquant.advanced
+            #if $cxt.excludemods != ''
+                #set $wfdict['ionquant.excludemods'] = $cxt.excludemods
+            #end if
+            #if $cxt.minscans != ''
+                #set $wfdict['ionquant.minscans'] = $cxt.minscans
+            #end if
+            #if $cxt.minisotopes != ''
+                #set $wfdict['ionquant.minisotopes'] = $cxt.minisotopes
+            #end if
+            #if $cxt.locprob != ''
+                #set $wfdict['ionquant.locprob'] = $cxt.locprob
+            #end if
+            #if $cxt.writeindex != 'None'
+                #set $wfdict['ionquant.writeindex'] = $cxt.writeindex
+            #end if
+        #elif $prfx.label_free_quantification_run == 'freequant'
+            #set $wfdict['quantitation.run-label-free-quant'] = 'false'
+            #set $wfdict['ionquant.run-ionquant'] = 'false'
+            #set $wfdict['freequant.run-freequant'] = 'true'
+            #set $cxt = $prfx.freequant
+            #if $cxt.rt_tol != ''
+                #set $wfdict['freequant.rt-tol'] = $cxt.rt_tol
+            #end if
+            #if $cxt.mz_tol != ''
+                #set $wfdict['freequant.mz-tol'] = $cxt.mz_tol
+            #end if
+        #end if
+    ]]></token>
+
+    <!--
+      Isobaric quantification options
+
+      TMT-n uses the following n first labels from TMT-18
+      TMT-18 "126, 127N, 127C, 128N, 128C, 129N, 129C, 130N, 130C, 131N, 131C, 132N, 132C, 133N, 133C, 134N, 134C, 135N"
+      iTRAQ-4 "114, 115, 116, 117"
+      iTRAQ-8 "113, 114, 115, 116, 117, 118, 119, 121"
+    -->
+    <xml name="isobaric_quantification" token_expand="false" token_yes="true" token_no="false" token_plex="" token_level="2" token_virtual="false" token_pool="false" token_bridge="true">
+        <section name="quant_iso" expanded="@EXPAND@" title="Quant (Isobaric)">
+            <conditional name="isobaric_quantification">
+                <param name="isobaric_quantification_step" type="select" label="Perform Isobaric Quantification">
+                    <option value="default">Use workflow defaults</option>
+                    <option value="yes" selected="@YES@">yes</option>
+                    <option value="no" selected="@NO@">no</option>
+                </param>
+                <when value="default"/>
+                <when value="no"/>
+                <when value="yes">
+                    <section name="options" expanded="true" title="Isobaric Quantification">
```

Comment:
Does having a single section make sense here?

Reply:
A little confused on this point. Would it be to remove the "no" and "default" options? The "yes"/"no" distinction allows us to set the param that enables this in FragPipe. The "default" is usually a bit redundant but the argument could be that if a user is re-running a shared job, they might not know the workflow default and could use that option to revert to it.

Reply:
default / no / yes is fine for me. was just wondering if a single section inside a conditional is helpful. 

but fine with me.